### PR TITLE
Feature/get and display flies

### LINKED ISSUE #34: User's Fly Box - UI
Happy Path:
As a user, when I load the app, I should see the home page which is my "fly box" and displays all flies I have saved.

Sad Path:
As a user, when I load the app, I should see the home page which is my "fly box" and displays a message telling me to add flies if none are found. 

- [x] [Create Fly Box Page](https://github.com/my-fly-box/my-fly-box-ui/issues/35)

- [x] [Create API request for user's flies](https://github.com/my-fly-box/my-fly-box-ui/issues/36)

- [x] [Load flies onto page](https://github.com/my-fly-box/my-fly-box-ui/issues/37)

- [x] [Display message if no flies found](https://github.com/my-fly-box/my-fly-box-ui/issues/38)

### LINKED ISSUE #36: Create API request for user's flies
Write the API request to send to the BE to retrieve a user's flies

### LINKED ISSUE #37: Load flies onto page
If the API request returns back saved flies, these should be displayed in order of createdAt date.

### LINKED ISSUE #38: Display message if no flies found
If the API request does not return back any saved flies, a message should be displayed informing the user that they do not have any saved flies and that they should add one.

### PR DESCRIPTION
**Pull request by rachael-t**
### Issue(s) related to this PR:
* closes #34
* closes #36 
* closes #37
* closes #38
 
 
 ### Type of change made:
* [ ] Bugfix (non-breaking change which fixes an issue)
* [X] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] Styling- no new features

#### If this submission is a new feature:
* [X] Have you added an explanation of what your changes do?
* [X] Have you lint your code locally prior to submission?

### Change Log:
* ApiCalls.js -> added the fetchFlies function to get the data on the server
* FlyBox.js -> imported fetchFlies and invoke the function under componentDidMount. Set the state for currentFlies with the resolved response object
* Fly.js -> make edits to props displaying the information

### Were there any challenges that arose? If so, how were the addressed?
* Yes, we ran into issues regarding CORs. Researched on the front-end to see if the issue was with the fetch request, but discovered it was something needing to be added to the server. Addressed by the backend team in https://github.com/my-fly-box/my-fly-box-api/issues/22 and resolved in https://github.com/my-fly-box/my-fly-box-api/pull/21

### Technical Debt:
* None

### Testing:
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] All new and existing tests passed?
* [ ] Have you run linter?

**Test coverage** = 0

### CODE REVIEW

File: ApiCalls.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  try {
+    const response = await fetch(
+      "https://my-fly-box-api.herokuapp.com/api/v1/flies"
+    );
+    const flyData = await response.json();
+    return flyData;
+  } catch (error) {
+    console.error(error);
+  }
+};
```
Comment (TimNguyen21) [inline ApiCalls.js:11]: This is good that you had made the fetch calls in its own file.  Now I can't incorporate this file to break out the fetch call in the Add Fly Component.  Thanks for setting this up!

File: containers/FlyBox.js [hunk 2]
```diff
@@ -1,133 +1,100 @@
...
+  constructor(props) {
+    super(props);
+  }
 
-    async componentDidMount() {
-        this.props.setFlies(TestFlies)
-    }
+  async componentDidMount() {
+    fetchFlies().then((data) => this.props.setFlies(data.data));
+  }
```
Comment (TimNguyen21) [inline containers/FlyBox.js:21]: This set up is perfect to store live data from the FlyBox API.  Now I believe we are in a good position to continue other iteration to reach our MVP.

Comment (rachael-t) [reply containers/FlyBox.js:21]: Agreed - it was nice and straight forward so should be helpful with the other aspects of our MVP

Comment (C-Bandstra) [reply containers/FlyBox.js:21]: Thank you for getting this done! now the project begins!!

**Pull request merged**